packages/calcite-components/src/components/progress/progress.tsx

```diff
@@ -34,7 +34,7 @@ export class Progress extends LitElement {
    */
   @property({ reflect: true }) type: "indeterminate" | "determinate" = "determinate";
 
-  /** When `type` is `"determinate"`, the component's progress value with a range of 0.0 - 1.0. */
+  /** The component's value. Valid only for `"determinate"` indicators. Percent complete of 100. */
```

Comment:
+@geospatialem @DitwanP for editorial perusal.

Reply:
WDYT about:

```suggestion
  /** When `type` is `"determinate"`, specifies the component's value with a range of 0 to 100. */
```